diff --git a/debian/rules b/debian/rules
old mode 100755
new mode 100644
index 5c58737..bc03c62
--- a/debian/rules
+++ b/debian/rules
@@ -4,6 +4,6 @@
 	dh $@

 override_dh_auto_configure:
-	dh_auto_configure -- --disable-readline
+	dh_auto_configure -- --disable-openssl

 override_dh_auto_test:
diff --git a/debian/patches/fix-include b/debian/patches/fix-include
new file mode 100644
index 0000000..96c7412
--- /dev/null
+++ b/debian/patches/fix-include
@@ -0,0 +1,12 @@
+Fix missing include.
+
+--- a/nestlex.c	2016-05-17 17:10:01.200441730 +0300
++++ b/nestlex.c	2016-05-17 17:10:47.512404942 +0300
+@@ -5,6 +5,7 @@
+ /* a function for lexical scanning of nested character patterns */
+
+ #include "config.h"
++#include <stddef.h>
+ #include "mytypes.h"
+
+ #include "sysincludes.h"
